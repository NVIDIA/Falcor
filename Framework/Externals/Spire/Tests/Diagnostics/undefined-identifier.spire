// use of undefined identifier

void foo()
{
	int a = b;
}
