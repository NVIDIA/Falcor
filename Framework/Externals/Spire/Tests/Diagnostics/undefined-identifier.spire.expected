result code = -1
standard error = {
Tests/Diagnostics/undefined-identifier.spire(5): error 30015: undefined identifier 'b'.
}
standard output = {
}
